src/libstd/thread/mod.rs

```diff
@@ -27,11 +27,15 @@
 //!
 //! Fatal logic errors in Rust cause *thread panic*, during which
 //! a thread will unwind the stack, running destructors and freeing
-//! owned resources. Thread panic is unrecoverable from within
-//! the panicking thread (i.e. there is no 'try/catch' in Rust), but
-//! the panic may optionally be detected from a different thread. If
-//! the main thread panics, the application will exit with a non-zero
-//! exit code.
+//! owned resources. While not meant as a 'try/catch' mechanism, panics
+//! in Rust can nonetheless be caught with
+//! ['catch_unwind'](../../std/panic/fn.catch_unwind.html) and recovered
```

Comment:
Better use backquotes (`` `…` ``) here (same for the `resume_unwind` below).

Reply:
Good point, thanks. Will update in a bit
The wording is otherwise sounding OK?

Reply:
Another suggestion :) This assumes that they can *always* be caught, but with panic=abort, they cannot still.

Reply:
Also a very good point :) Thanks